fiftyone/utils/depth_anything.py

```
@@ -0,0 +1,314 @@
+        img_array = img_array * 255
+    return np.clip(img_array, 0, 255).astype(np.uint8)
+
+
+class DepthAnythingV3OutputProcessor(fout.OutputProcessor):
+    """Output processor for Depth Anything V3 models.
+
+    Converts raw depth predictions to normalized
+    :class:`fiftyone.core.labels.Heatmap` instances.
+    """
+
+    def __call__(self, output, image_sizes, **kwargs):
```

Ruff (0.14.10)

Missing return type annotation for special method `__call__`

(ANN204)